bin/http-server

```diff
@@ -80,6 +82,54 @@ else if (colors) {
     request: function () {}
   };
 }
+if (superverbose) {
+  logger = {
+    info: console.log,
+    request: function (req, res, error) {
+      var date = utc ? new Date().toUTCString() : new Date();
+      if (error) {
+        logger.info(
+          '[%s] "%s %s" Error (%s): "%s"',
+          date, colors.red(req.method), colors.red(req.url),
+          colors.red(error.status.toString()), colors.red(error.message)
+        );
+        logger.info('---------------header--------------');
+        var  request = req.request;
+        var rawHeaders = request.rawHeaders;
+        if (rawHeaders && rawHeaders.length > 0) {
+          var headerStr = '';
+          for (var i = 0; i < rawHeaders.length;i++) {
+            headerStr += rawHeaders[i];
+            if (i % 2 === 0) {
+              headerStr += ': ';
+            }
+            else {
+              headerStr += '\n';
+            }
+          }
+          logger.info(colors.cyan(headerStr));
+        }
+        if (req.readable) {
+          var body = [];
+          req.request.on('data', function (chunk) {
+            body.push(chunk);
+          }).on('end', function () {
+            body = Buffer.concat(body).toString();
+            logger.info('-------------request body----------');
+            logger.info(body) ;
+          });
+        }
+      }
+      else {
```

Comment:
Since this is "superverbose", is there more information we can show if there was no error?